filebeat/module/auditd/log/ingest/pipeline.json

```diff
@@ -71,15 +71,18 @@
         },
         {
             "convert": {
-                "field" : "auditd.log.sequence",
+                "field": "auditd.log.sequence",
                 "type": "integer",
                 "ignore_missing": true
             }
         },
         {
             "script": {
-                "lang": "painless",
```

Comment:
It seems you removed  `lang: painless`. Was this on purpose?

Reply:
Painless is the default language so this field is unnecessary. However, better to be explicit so I'll add it back now.

Reply:
Yep, added the `lang` field back in c13108f.